### PR TITLE
Boo913561

### PR DESCRIPTION
**Pull request by DimStar77**
Let's start changing more things to Tumbleweed

### CODE REVIEW

File: control/control.openSUSE.xml [hunk 1]
```diff
@@ -143,36 +143,36 @@ textdomain="control"
...
-                <name>openSUSE-Factory-Debug</name>
+                <name>openSUSE-Tumbleweed-Debug</name>
                 <prod_dir>/</prod_dir>
                 <enabled config:type="boolean">false</enabled>
                 <autorefresh config:type="boolean">true</autorefresh>
                 <priority config:type="integer">99</priority>
             </extra_url>
             <extra_url>
-                <baseurl>http://download.opensuse.org/source/factory/repo/oss/</baseurl>
+                <baseurl>http://download.opensuse.org/source/tumbleweed/repo/oss/</baseurl>
```
Comment (kobliha) [inline control/control.openSUSE.xml]: http://download.opensuse.org/source/tumbleweed/repo/oss/

Object not found!

The requested URL was not found on this server. If you entered the URL manually please check your spelling and try again.

If you think this is a server error, please contact the webmaster.

Error 404

download.opensuse.org
Wed Jan 28 17:57:46 2015
Apache/2.2.12 (Linux/SUSE)

**Comment by kobliha** [comment]
One Error 404 otherwise looks good

File: control/control.openSUSE.xml (hunk 1 above)
Comment (DimStar77) [inline control/control.openSUSE.xml]: On Wed, 2015-01-28 at 08:58 -0800, Lukas Ocilka wrote:
> In control/control.openSUSE.xml:
> 
> >                  <prod_dir>/</prod_dir>
> >                  <enabled config:type="boolean">false</enabled>
> >                  <autorefresh config:type="boolean">true</autorefresh>
> >                  <priority config:type="integer">99</priority>
> >              </extra_url>
> >              <extra_url>
> > -                <baseurl>http://download.opensuse.org/source/factory/repo/oss/</baseurl>
> > +                <baseurl>http://download.opensuse.org/source/tumbleweed/repo/oss/</baseurl>
> 
> http://download.opensuse.org/source/tumbleweed/repo/oss/
> 
> Object not found!

Thanks, good catch

> The requested URL was not found on this server. If you entered the URL
> manually please check your spelling and try again.
> 
> If you think this is a server error, please contact the webmaster.

Will do.. this is definitively a server error :) 


-- 
Dimstar / Dominique Leuenberger <dimstar@opensuse.org>

Comment (DimStar77) [inline control/control.openSUSE.xml]: On Wed, 2015-01-28 at 08:58 -0800, Lukas Ocilka wrote:
> In control/control.openSUSE.xml:
> 
> >                  <prod_dir>/</prod_dir>
> >                  <enabled config:type="boolean">false</enabled>
> >                  <autorefresh config:type="boolean">true</autorefresh>
> >                  <priority config:type="integer">99</priority>
> >              </extra_url>
> >              <extra_url>
> > -                <baseurl>http://download.opensuse.org/source/factory/repo/oss/</baseurl>
> > +                <baseurl>http://download.opensuse.org/source/tumbleweed/repo/oss/</baseurl>
> 
> http://download.opensuse.org/source/tumbleweed/repo/oss/
> 
> Object not found!

This 404 has been fixed by now.
-- 
Dimstar / Dominique Leuenberger <dimstar@opensuse.org>

**Comment by kobliha** [comment]
OK, it's fixed, so good to go. LGTM

**Pull request merged**